tests/node-tests/strip-data-test-properties-plugin-test.js

```diff
@@ -3,12 +3,12 @@ let assert = require('assert');
 
 let babel7 = require('@babel/core');
 
-let StripDataTestPropertiesPlugin6 = require('../strip-data-test-properties-plugin6');
+let StripDataTestPropertiesPlugin6 = require('ember-test-selectors/strip-data-test-properties-plugin6');
 
 function testFixture(name) {
   it(`fixture: ${name}`, function () {
-    let fixturePath = `${__dirname}/fixtures/${name}/fixture.js`;
-    let expectedPath = `${__dirname}/fixtures/${name}/expected7.js`;
+    let fixturePath = `${__dirname}/fixtures/${name}/fixture.mjs`;
+    let expectedPath = `${__dirname}/fixtures/${name}/expected7.mjs`;
```

Comment:
Was that a necessary change? I'm not sure about this but if this somehow tests the plugin output, would that mean that older node versions won't be supported anymore because of the `,mjs` extension?

I guess the question is, are users affected by the extension change somehow?

Reply:
I think this point is interesting because we say the package is compatible with Node 12 but our CI check the compatibility starting Node 18. I think we shouldn't say we support a version that is not explicitly tested by the CI.